README.md

```diff
@@ -47,22 +47,19 @@ Integration tests require [`Wasmtime`](https://wasmtime.dev/) to be installed (`
 Before tests can be run, the cases must first be built, and then they can be tested:
 
 ```bash
-cmake --build cmake-build-debug --parallel 8 --target test-cases
 CTEST_OUTPUT_ON_FAILURE=1 make -C cmake-build-debug test
 ```
 
 Individual tests can also be run from within the build folder using `ctest` directly:
 
 ```bash
-cd cmake-build-debug
-ctest smoke
+ctest --test-dir cmake-build-debug smoke
```

Comment:
These instructions _almost_ work, but not entirely:
1. `ctest` seems to only work after running `make -C cmake-build-debug test`. Is there a way to address that? If not, I think it'd be good to keep a build target for the tests that doesn't also run them
2. working around the above, I can get `ctest` to run, but it seems to always run all tests, instead of just the `smoke` test as you indicate here. Should this use `-R [pattern]` instead?

Separately, I think it'd be good to eventually support loading the tests at runtime instead of having to wizen every test individually. That's what the WPT harness does as well. We shouldn't use this for all tests, and ideally make it an option, but for most tests I think that'd be fine, and speed things up considerably.

Reply:
1. `ctest` doesn't require `make -C cmake-build-debug test` to run first, just a normal `cmake --build cmake-build-debug`. I tested this again on a fresh build and it's definitely working.
2. Yes you're right about the `-R` pattern I've updated the example to note this.

Yes, runtime bulk tests are good, but also these tests can now catch all output and logs so are useful for fully isolated end to end test runs.

Perhaps the WPT tests and all the other unit tests can run as a single VM that just has everything for the fastest test run, and then we restrict these tests to only those that need to exercise the full end to end expectations.